clevertap-core/src/main/java/com/clevertap/android/sdk/inapp/images/Extensions.kt

```diff
@@ -13,4 +13,6 @@ fun File?.hasValidBitmap() : Boolean {
     options.inJustDecodeBounds = true
     BitmapFactory.decodeFile(this.path, options)
     return options.outWidth != -1 && options.outHeight != -1
-}
\ No newline at end of file
+}
+
+fun String?.isNotNullAndEmpty() : Boolean = isNullOrEmpty().not()
```

Comment:
This function will be better resided in the file `clevertap-core/src/main/java/com/clevertap/android/sdk/CTXtensions.kt`